base/multidimensional.jl

```diff
@@ -267,9 +267,6 @@ end
     _checksize(A, dim+1, J...)
 end
 
-@inline unsafe_setindex!(v::BitArray, x::Bool, ind::Int) = (Base.unsafe_bitsetindex!(v.chunks, x, ind); v)
-@inline unsafe_setindex!(v::BitArray, x, ind::Real) = (Base.unsafe_bitsetindex!(v.chunks, convert(Bool, x), to_index(ind)); v)
```

Comment:
Er, no, I was wrong — it's fine to delete these without expanding `setindex!` since the abstract fallbacks take care of the `to_index`ing these days.  This is ok.